meta/3rd/Defold/config.lua

```diff
@@ -0,0 +1,9 @@
+name    = 'Defold'
+files   = {'game.project'}
```

Comment:
This name is too common

Reply:
Hi!
Here is the empty project, in defold engine, 
also we can associate with file extensions .gui_script, .script, .collection, .render_script, .input_binding, 
but in fact on start we have always only one .project in root and one or more .collection in subfolders 
[MyProject123.zip](https://github.com/sumneko/lua-language-server/files/9955850/MyProject123.zip)

Reply:
maybe we can 
use one of this words in .project file?

[bootstrap]
main_collection = 

[project]
title =